docs/android/v2/quickstart/prebuilt-android.mdx

```diff
@@ -0,0 +1,87 @@
+---
+title: Prebuilt Quickstart
+nav: 5.3
+---
+
+## Overview
+
+This guide will walk you through the steps to create a video conferencing app using Prebuilt components of the `room-kit` package.
+Prebuilt is currently available for Android applications.
+
+> If you get stuck at any point or need help in understanding a concept, you can put your query in our [Discord Channel](https://100ms.live/discord).
+
+
+## Prerequisites
+
+-   _Android_: A basic familiarity with Android is required to follow this guide.
+-   _100ms account_: You need to have a 100ms account to create a room and get a `room-code`. You can create a 100ms account [here](https://dashboard.100ms.live/signup).
+-   _Room-code_: Prebuilt components require a `room-code` to join a room. You can get a `room-code` for a room using the 100ms dashboard. Refer to [this guide](/get-started/v2/get-started/prebuilt/room-codes/overview) to learn more about room codes.
+
+## Start a New Android Project (or use an existing one)
+[Here's how](https://developer.android.com/studio/projects/create-project) to create a new Android project.
+
+
+## Install Roomkit
+
+The `room-kit` library is available on Maven. Add it your app level build.gradle in the dependencies block.
+
+Latest Version:
+
+[![](https://img.shields.io/badge/dynamic/xml.svg?label=100ms-Prebuilt&color=blue&query=%2F%2Fmetadata%2Fversioning%2Flatest&url=https%3A%2F%2Frepo1.maven.org%2Fmaven2%2Flive%2F100ms%2Froom-kit%2Fmaven-metadata.xml)](https://jitpack.io/#100mslive/room-kit)
+
+```groovy
+dependencies { ....
+    implementation "live.100ms:room-kit:$GET_VERSION_FROM_ABOVE_BADGE"
+}
+```
+
+## Add the Prebuilt Component
+
+Look up how to get room codes in [this guide](/get-started/v2/get-started/prebuilt/room-codes/overview).
```

Comment:
```suggestion
Look up how to get `room code` in [this guide](/get-started/v2/get-started/prebuilt/room-codes/overview).
```

Reply:
In this context I believe it should be codes plural, but if you really think this is the right to refer to it in the doc I'll apply this.

Reply:
so if this this a quick start guide .. right now the developer / engineer is looking for a room code to instantiate the Prebuilt class. Not sure what will the engineer do with room codes ..